bundles/binding/org.openhab.binding.comfoair/README.md

```diff
@@ -1,458 +1,1020 @@
-        }
-        // Sonntag
-        else if( day == 7 ){
-            if( hour < 9 || (hour >= 22 && minute >=30) || hour >= 23 ) isNight = true
-        }
-        else{
-
-            if( hour < 7 || (hour >= 22 && minute >=30) || hour >= 23 ) isNight = true
-        }
-
-        var Number currentLevel = (Lueftung_Fan_Level.state as DecimalType)
-        var Number newLevel = 3
-
-        if( !isNight ){
-
-            var Number raumTemperatur = (Lueftung_Ablufttemperatur.state as DecimalType)
-            var Number aussenTemperatur = (Lueftung_Aussenlufttemperatur.state as DecimalType)
-            var Number zielTemperatur = (Lueftung_Komfortemperatur.state as DecimalType)
-
-            if(
-                raumTemperatur >= zielTemperatur
-                &&
-                aussenTemperatur >= raumTemperatur
-            ){
-
-                newLevel = 1
-            }
-            else if(
-                raumTemperatur >= zielTemperatur - 1
-                &&
-                aussenTemperatur >= raumTemperatur - 1
-                &&
-                currentLevel == 1
-            ){
-
-                newLevel = 1
-            }
-        }
-        else if( Fenster_OG_Schlafzimmer.state == CLOSED || Fenster_OG_Ankleide.state == OPEN ) {
-
-            newLevel = 2
-        }
-
-        if( newLevel != currentLevel ){
-
-            if( newLevel == 1 ){
-
-                logInfo("airflow.rules", "auto slowdown start")
-                    send("test@gmail.com", "Lüftung verlangsamt")
-            }
-            else if( currentLevel == 1 ){
-
-                    logInfo("airflow.rules", "auto slowdown end")
-                send("test@gmail.com", "Lüftung wieder normal")
-            }
-
-            autoChangeInProgress=true
-            sendCommand(Lueftung_Fan_Level,newLevel)
-        }
-    }
-end
-```
+# ComfoAir Binding modified
+Expanded ComfoAir binding v.1x for openHAB based on original [org.openhab.binding.comfoair](https://github.com/openhab/openhab1-addons/wiki/comfo-air-binding) v. 1.3.0 written by Holger Hees.
+
+## What's new in this binding ?
+
+Many new commands (read and write) have been added in this binding. The program code was also changed so the sending of the write commands is possible and did not cause problems with the operation of the device.<br/>
+Original binding provided 22 commands, including 5 write commands only (eg it allowed to control ventilation levels and reset errors).
+This new binding provides 160 commands, including 71 write commands. This gives you the ability to control all ComfoAir settings, available in menus 10 - 90, and even more. This allows for full control of the device.
+
+This binding was tested with the Zehnder ComfoAir 350 ventilation system. As original binding it should work with ComfoAir 550, WHR930 of StorkAir, G90-380 by Wernig and Santos 370 DC to Paul.
+
+### List of commands (original and new ones).

```

Comment:
Remove this section and put the table contents back into the original table in the "Binding Configuration" section.

Reply:
It was moved to "Item Configuration" section as in original readme.md file